From 8cd98c008dce9ddd406803215e50fddaa503ce0b Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Sat, 10 Feb 2018 20:10:48 -0500
Subject: [PATCH 06/27] fix ctrlport install

GR_PYTHON_INSTALL isn't needed in Debian systems. Using standard
CMake install does the right thing.
---
 gnuradio-runtime/python/gnuradio/ctrlport/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnuradio-runtime/python/gnuradio/ctrlport/CMakeLists.txt b/gnuradio-runtime/python/gnuradio/ctrlport/CMakeLists.txt
index f40f253a72..d717dbad1d 100644
--- a/gnuradio-runtime/python/gnuradio/ctrlport/CMakeLists.txt
+++ b/gnuradio-runtime/python/gnuradio/ctrlport/CMakeLists.txt
@@ -28,7 +28,7 @@ install(
 )
 
 
-GR_PYTHON_INSTALL(
+install(
     FILES
     ${CMAKE_CURRENT_SOURCE_DIR}/__init__.py
     ${CMAKE_CURRENT_SOURCE_DIR}/GrDataPlotter.py
-- 
2.11.0

